pkg/store/filenames/filenames.go

```diff
@@ -42,6 +42,8 @@ const (
 	HostAgentSock      = "ha.sock"
 	HostAgentStdoutLog = "ha.stdout.log"
 	HostAgentStderrLog = "ha.stderr.log"
+	Identifier         = "identifier"
```

Comment:
?

Reply:
Please describe in https://github.com/lima-vm/lima/blob/master/docs/internal.md

Also, perhaps the filename should be something like `vz-identifier`?

Reply:
Done